### ISSUE TITLE
Feature Request: Filter Out Duplicates

### ISSUE THREAD

**Issue by eightyy8** [Ranchero-Software/NetNewsWire#3542]
To stop repeated stories from multiple different sources and to clean up your smart feeds, have a duplicate filter option; edit wich sources are superior in certain categories, or hide stories from a certain source if another uses the same phrases. I recommend placing it inside the unread button and making a tray to hold all the filtering options upon opening.

**Comment by djjerguthrie** [comment]
Or even just collapse articles of the same title under a ‘folder’ into one entry and just pick one to use at random in the collapsed item.  e.g. four feeds from the NYT for different subjects might all have the same article exploding counts.

**Comment by brentsimmons** [comment]
I totally understand wanting this feature. There are two big issues with it:

1. Performance would be abysmal. Checking every new article against every existing article would take a very long time.
2. There might be some shortcuts, yes, that might perform better — like just looking at titles — but this will have false positives and, more importantly, it will never really satisfy. People would still have near-duplicates that they’d like filtered out.

In other words, I believe there is no option that both performs well and does what people actually want.
